### DISCUSSION TITLE
[Piano Notes] (No JSON) His Theme (Undertale)

### DISCUSSION THREAD

**Discussion by Abc2557** [Monika-After-Story/MonikaModDev#7798] [general - piano notes]
Here are the notes for His Theme from the video game Undertale (same tune as 'Memory' and 'Hopes and Dreams,' two other songs from Undertale):
W Y T W R R T
W T W R R T
W Y T W R R T 
W T U Y T Y
W Y T W R R T
W T W R R T
W Y T W R R T 
W T U Y T Y
W O Y
T O W
W T O P O Y T
W O U Y T R T Y
[ P O
T O U Y T R T R
Y Y T R T W
T O P O Y T
W O U Y T R T Y
[ P O
T O U Y T R T R
Y Y T R T
W Y T W R R T
W T W R R T
W Y T W R R T 
W T U Y T Y
W Y T W R R T
W T W R R T
W Y T W R R T

Sorry that it's so long! I wanted to include most of the tunes in the song.
Good luck to the person who decides to make the JSON for this!

**Comment by Iniquitatis** [comment]
I've already thought about writing converter from such format to JSON, but maybe it would be even more convenient to generate it directly from the MIDI data.

**Comment by Iniquitatis** [comment]
There are no lyrics, so blank text bar will appear while playing. Still, it might be a good starting point.
[Boop](https://github.com/Monika-After-Story/MonikaModDev/files/4574405/histheme.txt) (rename `.txt` to `.json` and put it into the `piano_songs` directory).

**Comment by Abc2557** [comment]
Thank you!
